diffraction/WISH/bragg-detect/cnn/BraggDetectCNN.py

```diff
@@ -37,28 +45,75 @@ def __init__(self, model_weights_path, batch_size=64, workers=0, iou_threshold=0
         self.iou_threshold = iou_threshold
 
 
-    def find_bragg_peaks(self, workspace, output_ws_name="CNN_Peaks", conf_threshold=0.0, q_tol=0.05):
+    def find_bragg_peaks(self, workspace, output_ws_name="CNN_Peaks", conf_threshold=0.0, clustering=Clustering.QLab.name, **kwargs):
         """
         Find bragg peaks using the pre trained FasterRCNN model and create a peaks workspace
         :param workspace: Workspace name or the object of Workspace from WISH, ex: "WISH0042730"
         :param output_ws_name: Name of the peaks workspace
         :param conf_threshold: Confidence threshold to filter peaks inferred from RCNN
-        :param q_tol: qlab tolerance to remove duplicate peaks
+        :param clustering: name of clustering method. Default is QLab and allowed
+        :param kwargs: variable keyword params for clustering methods
```

Comment:
Can you add details for where a list of kwargs can be found - e.g. reference to HDBCSAN docs and `BaseSX.remove_duplicate_peaks_by_qlab`?